app/helpers/audit_event_helper.rb

```diff
@@ -0,0 +1,41 @@
+module AuditEventHelper
+  def audit_create_item(item)
+    create_audit_event(item, :create_item)
+  end
+
+  def audit_request_lend(item)
+    create_audit_event(item, :request_lend)
+  end
+
+  def audit_accept_lend(item)
+    create_audit_event(item, :accept_lend)
+  end
+
+  def audit_request_return(item)
+    create_audit_event(item, :request_return)
+  end
+
+  def audit_accept_return(item)
+    create_audit_event(item, :accept_return)
+  end
+
+  def audit_deny_return(item)
+    create_audit_event(item, :deny_return)
+  end
+
+  def audit_add_to_waitlist(item)
+    create_audit_event(item, :add_to_waitlist)
+  end
+
+  def audit_leave_waitlist(item)
+    create_audit_event(item, :leave_waitlist)
+  end
+
+  private
+
+  def create_audit_event(item, event_type)
+    audit_event = AuditEvent.new(item: item, owner_id: item.owner, holder_id: item.holder,
+                                 triggering_user: current_user, event_type: event_type)
```

Comment:
Currently, we are not really using the fact that `holder` could be `null` maybe we should filter depending on the event if we want a holder to be `present`. For example, we don't need a holder for *waitlist events*.